01-node-tutorial/12-http.js

```diff
@@ -21,11 +21,11 @@ const server = http.createServer((req, res) => {
   if (req.url === '/') {
     res.end('Welcome to our home page')
   } else if (req.url === '/about') {
-    res.end('Here is our short history')
+    res.end('Here is our history')
   } else {
     res.end(`
     <h1>Oops!</h1>
-    <p>We can't seem to find the page you are looking for</p>
+    <p>We can't to find the page you are looking for</p>
     <a href="/">back home</a>
     `)
   }
```

Comment:
Nice implementation but I spotted some comments at the beginning of this function which I believe are not necessary here. check this out [comments best practice](https://swimm.io/learn/code-collaboration/comments-in-code-best-practices-and-mistakes-to-avoid)